src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Ui {
+    private final static String UNDERLINE = "_________________________________";
+
+    /**
+     * Outputs the greeting for Duke
+     *
+     */
+    public static String intro() {
+        String res = UNDERLINE + "\n" + "Hello! I'm Duke!"  + "\n" + "What can I do for you?" + "\n" + UNDERLINE;
```

Comment:
Maybe for the repetitive UNDERLINE used in this file, you can use a function to format all the output with that?